paikkala/models/tickets.py

```diff
@@ -1,12 +1,14 @@
+import datetime
```

Comment:
`from datetime import datetime` for less noise?

Reply:
Nix – I like to keep `datetime` imports like this because it's way too easy to get a `module object is not callable` or vice versa.